db/db_impl/db_impl.cc

```diff
@@ -1927,11 +1927,18 @@ Status DBImpl::Get(const ReadOptions& read_options,
                    PinnableSlice* value, std::string* timestamp) {
   assert(value != nullptr);
   value->Reset();
+
+  ReadOptions complete_read_options(read_options);
+  if (complete_read_options.io_activity == Env::IOActivity::kUnknown) {
+    complete_read_options.io_activity = Env::IOActivity::kGet;
+  }
+
   GetImplOptions get_impl_options;
   get_impl_options.column_family = column_family;
   get_impl_options.value = value;
   get_impl_options.timestamp = timestamp;
-  Status s = GetImpl(read_options, key, get_impl_options);
+
+  Status s = GetImpl(complete_read_options, key, get_impl_options);
```

Comment:
[Design] The reason why I decided to change `read_options` above `GetImpl()` (or above `VerifyChecksumInternal()` is other db.h level API call (e.g, `  virtual bool KeyMayExist(const ReadOptions& options,
`)can also call `GetImpl()`